src/clr-core/.storybook/main.js

```diff
@@ -0,0 +1,30 @@
+const { TsConfigPathsPlugin } = require('awesome-typescript-loader');
+const path = require('path');
+
+module.exports = {
```

Comment:
This file allows us to configure how storybook processes our story files. This should be kept minimal and is just to allow us to write stories in TypeScript.